build.gradle

```diff
@@ -581,41 +542,6 @@ bootJar {
   }
 }
 
-configurations.all {
-  resolutionStrategy.eachDependency { details ->
-        // Remedy for CVE-2020-8908, ensures launchdarkly uses correct version
-        if (details.requested.group == 'com.google.guava'
-                && details.requested.name == 'guava') {
-            details.useVersion "32.1.1-jre"
-        }
-    }
-
-    resolutionStrategy.eachDependency { details ->
-        if (details.requested.group == 'org.bouncycastle'
-                && (details.requested.name == 'bcpkix-jdk18on'
-                || details.requested.name == 'bcprov-jdk18on')
-        ){
-            details.useVersion versions.bouncycastle
-        }
-    }
-
-  resolutionStrategy.eachDependency { details ->
-    // Remedy for CVE-2024-25710
-    if (details.requested.group == 'org.apache.commons'
-            && details.requested.name == 'commons-compress') {
-      details.useVersion "1.26.0"
-    }
-  }
```

Comment:
a run of:
`./gradlew dependencyInsight --no-single-path --rerun --configuration runtimeClasspath --dependency commons-compress`
reveals that 1.19 is being used. please restore resolution strategy as remedy due to CVE-2024-25710